Lib/test/test_ast/test_ast.py

```diff
@@ -2303,15 +2303,14 @@ def get_load_const(self, tree):
         co = compile(tree, '<string>', 'exec')
         consts = []
         for instr in dis.get_instructions(co):
-            if instr.opname == 'LOAD_CONST' or instr.opname == 'RETURN_CONST':
+            if instr.opname in ('LOAD_CONST', 'LOAD_CONST_IMMORTAL'):
                 consts.append(instr.argval)
         return consts
 
     @support.cpython_only
     def test_load_const(self):
         consts = [None,
                   True, False,
-                  124,
```

Comment:
Why was `124` removed? I think that we should back `124` here and just add `LOAD_INT` to tuple of possible bytecodes on line 2306

Reply:
I've put it back.